### ISSUE TITLE
Save on Compile/Run

### ISSUE THREAD

**Issue by gilcu3** [mfornet/acmx#15]
When running compile or run command, it would be nice if the source file is auto saved

**Comment by gilcu3** [comment]
something like this should work
```
vscode.window.activeTextEditor.document.save().then( () => {
// file saved
},
() => {
// error
});
 ```

**Comment by mfornet** [comment]
This is already done:
- https://github.com/mfornet/acmx/blob/master/src/extension.ts#L173
- https://github.com/mfornet/acmx/blob/master/src/extension.ts#L212

will be published on version 0.2.8
